deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,54 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+name: mycluster
```

Comment:
remove `name` attribute and replace it with `.Release.Name`?

Reply:
done. [7ca1f29](https://github.com/apecloud/kubeblocks/pull/3865/commits/7ca1f2976a54c848653289f6b7235150fb9ced69)